content/id/docs/kegiatan-eastern-townhall-indonesia.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Kegiatan Eastern Townhall Indonesia"
+linkTitle: "Kegiatan Eastern Townhall Indonesia"
+hide_feedback: true
+weight: 20
+# description:
+---
+
+#### Sebagai informasi, kegiatan ini khusus di Eastern Townhall (ETH) Indonesia saja dan dimulai sejak tanggal 10 Jan 2022
```

Comment:
do you want to put this into the description? or you can simply put it in as a note using italics up to you.

Reply:
Yeah, I agree. I will put it using italics.